lib/lib-dynamodb/src/commands/BatchGetCommand.ts

```diff
@@ -63,6 +64,13 @@ export class BatchGetCommand extends $Command<
     },
   ];
 
+  protected readonly clientCommand = new __BatchGetItemCommand(this.input as any);
```

Comment:
nit: It shouldn't matter, but set command in the constructor so that it improves code readability?
i.e. it creates command after input is set.

Example:
```ts
  // ...
  protected readonly clientCommand: __BatchGetItemCommand;
  // ...
  constructor(readonly input: BatchGetCommandInput) {
    super();
    this.clientCommand = new __BatchGetItemCommand(this.input as any);
  }
```